src/codecs/encrypted.js

```diff
@@ -0,0 +1,44 @@
+// @ts-check
+import * as varint from '../varint.js'
+import { codec } from './codec.js'
+
+const code = 0x1400
+
+/**
+ * @template {number} Code
+ * @param {Object} options
+ * @param {Uint8Array} options.bytes
+ * @param {Uint8Array} options.iv
+ * @param {Code} options.code
```

Comment:
What' the code here ? I'm bit confused because as far as I understand `0x1400` code for `encrypted` codec, but then there is this other `code` which I'm not quite sure what that one represents.

Some comments would be really helpful here

Reply:
maybe it should be `cipher` instead of `code`, that's what it's for, a generic encrypted block that uses an `iv` with the cipher specified by looking up this integer in the multicodec table